tests/test_socket.py

```diff
@@ -146,3 +146,24 @@ def test_client_socket_set_timeout():
     conn.close()
     client_socket.close()
     server_socket.close()
+
+
+def test_client_socket_not_open_after_failed_open():
+    client_socket = TSocket(host="localhost", port=12345, socket_timeout=100)
```

Comment:
you can omit the `socket_timeout=...` here and in the test below; i assume this was copy/paste from a test for which it did matter.